### DISCUSSION TITLE
What to do next after database is paused for more than 90 days

### DISCUSSION THREAD

**Discussion by Princewil** [supabase/supabase#30404] [General]
What do I do next??

Below is a picture

**Comment by Princewil** [comment]
<img width="1552" alt="Screenshot 2024-11-10 at 23 43 32" src="https://github.com/user-attachments/assets/e5a3836d-32ca-4587-b552-b7f2d2ffd915">

**Comment by GaryAustin1** [comment]
https://supabase.com/docs/guides/platform/upgrading#time-limits
